test/xspec.bats

```diff
@@ -326,3 +326,20 @@ teardown() {
 	grep '<style type="text/css">' ../tutorial/xspec/escape-for-regex-result.html
 	grep 'margin-right:' ../tutorial/xspec/escape-for-regex-result.html
 }
+
+
+@test "running XSpec via ant for XSLT support" {
+    run ant -buildfile ${PWD}/../build.xml -Dxspec.xml=${PWD}/../tutorial/escape-for-regex.xspec -lib /tmp/xspec/saxon/saxon9he.jar -lib /tmp/ant/lib/xmltask.jar -lib /tmp/ant/lib/ant-contrib.jar 
+	echo $output
+    [ "$status" -eq 0 ]
+    [[ "${output}" =~  "BUILD SUCCESSFUL" ]]
+}
+
+
+@test "running XSpec via ant for Schematron support" {
+    echo ${PWD}
+    run ant -buildfile ${PWD}/../build.xml -Dxspec.xml=${PWD}/../tutorial/schematron/demo-03.xspec -lib /tmp/xspec/saxon/saxon9he.jar -lib /tmp/ant/lib/xmltask.jar -lib /tmp/ant/lib/ant-contrib.jar -Dtest.type=s -Dxspec.project.dir=${PWD}/../ -Dxspec.compiled.xsl.dir=${PWD}/../tutorial/schematron -Dxspec.phase=#ALL
```

Comment:
After running this command line locally, `git status` finds `tutorial/schematron/demo-03-sch-compiled.xsl` even with `-Dclean.output.dir=true`. Is it a bug of `clean.output.dir`?

Reply:
I remember @markdunnoup mentioning this in https://github.com/xspec/xspec/pull/138#issue-236885420:

>The "cleanup" target is not fully compatible with XSpec for Schematron yet. When ${clean.output.dir} is set to true, it removes SVRL files which the HTML report links to if the XSPec fails.

Reply:
The problem here is that a file is *not* removed. The known problem in the cited comment is that a file *is* removed. Are they the same thing?
Anyway, if `cleanup` is incomplete, it would be nice if known issues or remaining tasks are commented in the file.

Reply:
I added the removal of the temporary file in https://github.com/xspec/xspec/commit/3b1a6f49032aef61941c06a5fd5bb6eeb82724d6#diff-2cccd7bf48b7a9cc113ff564acd802a8R252, the XSL compiled file was missing.